data/strings/strings.txt

```diff
@@ -26079,6 +26084,7 @@
     pt-BR = Não foi possível criar a pasta e mover os arquivos à memória interna ou sdcard do dispositivo
     ru = Не могу создать папку и переместить файлы на устройстве
     tr = Dahili aygıtın belleğinde veya SD kartta klasör oluşturulamıyor ve dosyalar taşınamıyor
+    uk = Використовується на екрані запуску DownloadResources
```

Comment:
```suggestion
    uk = Неможливо створити папку та перемістити файли до внутрішньої памʼяті або SD карти
```

You translated comment instead of actual text